templates/editConn.js

```diff
@@ -0,0 +1,76 @@
+const vscode = acquireVsCodeApi();
```

Comment:
Does this file have to be javascript, could it be typescript?

Reply:
Right, i initially did something like it. But seems vscode made some hacks for `acquireVsCodeApi`,  which results some complex dependency issue. I couldn't really find a working example, so I ended up like that.